source/features/show-asset-download-count.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {v4, escapeKey} from '../libs/api';
+import {cloudDownload} from '../libs/icons';
+import {getOwnerAndRepo, getRepoPath} from '../libs/utils';
+import {wrap} from '../libs/dom-utils';
+
+interface Asset {
+	name: string;
+	downloadCount: number;
+}
+interface Tag {
+	[key: string]: Asset[];
+}
+async function getAssetsForTag(tags: string[]): Promise<Tag> {
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const data = await v4(
+		'{' +
+			tags.map(tag =>
+				escapeKey(tag) + `: repository(owner: "${ownerName}", name: "${repoName}") { release(tagName:"${tag}") { releaseAssets(first: 100) { edges { node { name downloadCount } } } } }`
+			) +
+		'}'
+	);
+	const assets: Tag = {};
+	for (const [tag, repo] of Object.entries(data)) {
+		assets[tag] = repo.release.releaseAssets.edges.map((edge: any) => edge.node);
+	}
+
+	return assets;
+}
+
+async function init(): Promise<void | false> {
+	if (getRepoPath() === null || !(getRepoPath() as string).startsWith('releases')) {
+		return;
+	}
+
+	const tags = select.all('svg.octicon-tag ~ span').map(tag => tag.textContent!);
+	const tagAssets = await getAssetsForTag(tags);
+	for (const release of select.all('.release-entry:not(.release-timeline-tags)')) {
+		const tagName = escapeKey(select('svg.octicon-tag ~ span', release)!.textContent!);
+		if (!tagAssets[tagName]) {
```

Comment:
Does this actually happen? Can a whole asset list be missing or should it be an empty array?